core/controllers/suggestion_test.py

```diff
@@ -3222,8 +3222,9 @@ def test_handler_with_invalid_suggestion_type_raise_error(self) -> None:
         )
 
     def test_exploration_handler_returns_data_with_no_limit(self) -> None:
-        user_services.update_preferred_translation_language_code(
-            self.reviewer_id, 'hi')
+        user_settings = user_services.get_user_settings(self.reviewer_id)
+        user_settings.preferred_translation_language_code = 'hi'
+        user_services.save_user_settings(user_settings)
```

Comment:
Can you add new lines to differentiate setup, action and verification of this test? https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests

Reply:
Done

Reply:
Done.